src/fetch_agent/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>fetch_agent</name>
+  <version>0.0.0</version>
```

Comment:
If version tag is used, change version number on changes imo

Reply:
it's just being created by default.
i don't think there is any need for us to use it